### DISCUSSION TITLE
Not connected in v0.0.8

### DISCUSSION THREAD

**Discussion by cbuschbeck** [aturri/ha-zcsazzurro#15] [Q&A]
Requested Auth_Key and Clientcode, added it to the config and used HACS to get the integration. 

Debug shows following:
Unable to parse result from ZCS Azzurro portal: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>502 Proxy Error</title>
</head><body>
<h1>Proxy Error</h1>
<p>The proxy server received an invalid
response from an upstream server.<br />
The proxy server could not handle the request<p>Reason: <strong>Error reading from remote server</strong></p></p>
<hr>
<address>Apache/2.4.47 (Win64) OpenSSL/1.1.1k PHP/8.0.6 Server at third.zcsazzurroportal.com Port 19003</address>
</body></html>

**Comment by aturri** [comment]
Hi,
This is a server error, there is nothing possible to do on client side.
Sometimes, ZCS server is not reachable and you may see this error in logs because it is not possible to get data.
The integration will update data at regular intervals, as soon as the server is responding you can get data 😄
